docs/robots/warthog/user_manual_warthog.mdx

```diff
@@ -148,233 +233,122 @@ If you're not seeing any change in behaviour, please contact our support team.
-This is configured to set the `ROS_HOSTNAME` environment variable to the Warthog PC's hostname.
-If your network resolves hostnames properly, connecting should be a matter of executing the following two lines from your laptop (or sourcing a script containing these lines):
-
-```
-export ROS_MASTER_URI=http://cpr‐warthog:11311 # Your robot's hostname
-export ROS_IP=192.168.12.123 # Your computer's wireless IP address
-```
-
-If your network doesn't resolve hostnames, you may need to add the following line to your `/etc/hosts` file:
-
-```
-192.168.131.1 cpr‐warthog # The robot's wireless IP address and hostname
-```
-
-Once everything is set up correctly, try running `rostopic` list.
-This will verify that your machine can see the robot's ROS master.
-Then try running rostopic echo `/mcu/status`.
-This will verify that the robot PC can see your machine in order to stream topics to it.
-
-Please contact Clearpath Support if guidance is required in selecting and executing a remote access strategy.
-For more general details on how ROS works over TCP with multiple machines, please see:
-
-<http://wiki.ros.org/ROS/Tutorials/MultipleMachines>
-
-For help troubleshooting a multiple machines connectivity issue, see:
-
-<http://wiki.ros.org/ROS/NetworkSetup>
-
-### Visualizing Warthog
-
-To command or observe Warthog from your desktop computer, first set up a basic ROS installation.
-See the following page for details:
-
-<http://wiki.ros.org/noetic/Installation/Ubuntu>
-
-When your ROS install is set up, install the Warthog desktop packages:
-
-```
-sudo apt‐get install ros‐noetic‐warthog‐desktop
-```
-
-Once your remote access to Warthog's ROS master is configured (as above), you can launch RViz, the standard ROS robot visualization tool:
+The long range remote control (RC) Futaba radio transmitter can be used to teleoperate the Warthog.
+To begin, slide the power switch to the ON position which is labelled in the figure below.
 
-```
-roslaunch warthog_viz view_robot.launch
-```
+:::caution
 
-From within RViz, you can use interactive markers to drive Warthog.
-You can visualize its published localization estimate, and you can visualize any attached sensors which have been added to its robot description XML (URDF).
+The speed adjustment knob should be turned initially all the way to the left while familiarizing yourself with the
+transmitter and slowly increasing it to get it moving.
 
-From your laptop, you can also launch the standard RQT Robot Monitor. This will report the diagnostic output from Warthog's self-monitoring capabilities, as shown in the figure below:
+:::
 
-```
-rosrun rqt_robot_monitor rqt_robot_monitor
-```
+The position of the speed adjustment scale is shown in the figure as _CH4_.
+The transmitter needs to be enabled which is done using the _Deadman Switch_.
+The _Deadman Switch_ is a two position spring loaded switch.
```

Comment:
Note; new robot manuals should use _Enable Switch_ rather than _Deadman Switch_ to be more aligned with industry norms.

I don't suggest changing this for now.
This can be updated later after we make new Futaba joystick graphics.

Reply:
Updated, including the graphics